Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/ValueStringSort.cpp

```diff
@@ -38,11 +38,7 @@ namespace AZ::DocumentPropertyEditor
 
     RowSortAdapter::SortInfoNode* ValueStringSort::NewSortInfoNode() const
```

Comment:
suggestion: Might be out of scope of this PR, but as this is returning a pointer that is owning, it might be nice to update this to a `unique_ptr` to indicate the transfer of ownership and ensure people can't forget to delete it.